src/main/java/io/cdap/e2e/utils/CdfHelper.java

```diff
@@ -16,26 +16,115 @@
 
 package io.cdap.e2e.utils;
 
+import io.cdap.e2e.pages.actions.CdfGcsActions;
+import io.cdap.e2e.pages.actions.CdfLogActions;
+import io.cdap.e2e.pages.actions.CdfPipelineRunAction;
+import io.cdap.e2e.pages.actions.CdfStudioActions;
+import io.cdap.e2e.pages.locators.CdfGCSLocators;
+import io.cdap.e2e.pages.locators.CdfPipelineRunLocators;
+import org.junit.Assert;
+import org.openqa.selenium.By;
 import org.openqa.selenium.NoAlertPresentException;
-import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import stepsdesign.BeforeActions;
 
 import java.io.IOException;
-
-import static io.cdap.e2e.utils.RemoteClass.createDriverFromSession;
+import java.util.UUID;
 
 /**
  * Represents CdfHelper
  */
 public interface CdfHelper {
 
   default void openCdf() throws IOException, InterruptedException {
-    SeleniumDriver.openPage(SeleniumHelper.readParameters("CDFURL"));
+    SeleniumDriver.openPage(SeleniumHelper.readParameters(ConstantsUtil.CDFURL));
     try {
       SeleniumDriver.getDriver().switchTo().alert().accept();
       SeleniumDriver.waitForPageToLoad();
     } catch (NoAlertPresentException Ex) {
       SeleniumDriver.waitForPageToLoad();
     }
+  }
+
+  default int countOfNoOfRecordsTransferredToBigQueryIn(String records) throws IOException, InterruptedException {
```

Comment:
Instead of 'records', the input parameter will be 'tableName'

Reply:
done